src/mixxx.h

```diff
@@ -138,28 +119,22 @@ class MixxxMainWindow : public QMainWindow {
             SoundDeviceError err, bool* retryClicked);
     QDialog::DialogCode noOutputDlg(bool* continueClicked);
 
-    // Pointer to the root GUI widget
-    QWidget* m_pWidgetParent;
+    QWidget* m_pRootWidget;
```

Comment:
I think m_pCentralWidget; is more Qt like

Reply:
`m_pMainWidget` maybe?

Reply:
The comment clarified that it was the root widget, and I think @uklotzde said that we should replace such comments with descriptive naming, that's why I named it that way. I don't think the question about naming is what is "Qt like", but rather what conveys the necessary information to the developer?

Reply:
"Qt like" because 
setCentralWidget(m_pCentralWidget);
the variable holds a pointer to the current centralWidget

m_pMainWidget works also for me and since it is the opinion of the majority, just follow.

Reply:
Oh, I didn't realize the function was named like that. In that case I prefer m_pCentralWidget, too.

Reply:
Adjusted :)